srv/modules/runners/validate.py

```diff
@@ -1054,6 +1074,150 @@ def report(self):
         self.printer.print_result()
 
 
+class ConfigCheck(object):
+    """Class to detect deprecated config values in files.
+
+    Attributes:
+        base_path (str): Base path
+        map_file (str): Map file path
+        conf_path (str): Conf file path
+        suffix (str): Suffix for config files
+        files (list): List of files from glob
+        map (dict): Map of deprecated k:v
+        issues (list): List of found incidents
+    """
+    def __init__(self):
+
+        self.base_path = '/srv/salt/ceph/configuration/files'
+        self.map_file = '{}/deprecated_map.yml'.format(self.base_path)
+        self.conf_path = '{}/ceph.conf.d'.format(self.base_path)
+        self.suffix = '.conf'
+        self.files = glob.glob("{path}/*{suffix}".format(path=self.conf_path,
+                                                         suffix=self.suffix))
```

Comment:
Can we add something like the following (untested), to pick up the imported config file produced by engulf?

```
        if os.path.isfile("/srv/salt/ceph/configuration/files/ceph.conf.import"):
            files.append("/srv/salt/ceph/configuration/files/ceph.conf.import")
```